src/components/LayoutContainer.tsx

```diff
@@ -1,15 +1,15 @@
-import React, { useState, useEffect } from 'react';
-import { MainContent } from './MainContent.jsx';
-import { Sidebar } from './Sidebar/Sidebar.jsx';
+import { useState } from 'react';
+import { MainContent } from '../components/MainContent';
+import { Sidebar } from '../components/Sidebar/Sidebar';
```

Comment:
Cant we import from the same folder?

Reply:
Sure, we can also implement an index file and import everything from feature folder or just from /components root:

`import { MainContent } from './MainContent';
import { Sidebar } from './Sidebar';`
Or
`import { MainContent, Sidebar } from './';`

Reply:
Thanks, @Enzo707 Would you update? I would like the import statements to be consistent

Reply:
@QilongTang I already updated the imports. PTAL